src/main/resources/db/migration/V1_105__housekeeping_juror_deletion.sql

```diff
@@ -0,0 +1,271 @@
+			p_print_msg := 'ERROR:-> TIMED OUT';
+			EXIT Deletes;
+		END IF;
+
+		delete from juror_mod.juror_reasonable_adjustment ra where ra.juror_number = p_juror_number;
+		SELECT juror_mod.check_time_expired(p_start_time_int,p_max_timeout) INTO v_timed_out;
+		IF v_timed_out THEN
+			p_print_msg := 'ERROR:-> TIMED OUT';
+			EXIT Deletes;
+		END IF;
+
+		delete from juror_mod.juror_response_cjs_employment cjs where cjs.juror_number =  p_juror_number;
+		SELECT juror_mod.check_time_expired(p_start_time_int,p_max_timeout) INTO v_timed_out;
+		IF v_timed_out THEN
+			p_print_msg := 'ERROR:-> TIMED OUT';
+			EXIT Deletes;
+		END IF;
+
+        delete from juror_mod.juror_response_aud jra where jra.juror_number = p_juror_number;
+        SELECT juror_mod.check_time_expired(p_start_time_int,p_max_timeout) INTO v_timed_out;
+		IF v_timed_out THEN
+			p_print_msg := 'ERROR:-> TIMED OUT';
+			EXIT Deletes;
+		END IF;
+
+		delete from juror_mod.juror_response jr where jr.juror_number = p_juror_number;
+		SELECT juror_mod.check_time_expired(p_start_time_int,p_max_timeout) INTO v_timed_out;
+		IF v_timed_out THEN
+			p_print_msg := 'ERROR:-> TIMED OUT';
+			EXIT Deletes;
+		END IF;
+
+		-- Juror parent record
+		DELETE FROM juror_mod.juror j WHERE j.juror_number = p_juror_number;
+		-- check if timeout has elapsed - if so exit loop
+	    SELECT juror_mod.check_time_expired(p_start_time_int,p_max_timeout) INTO v_timed_out;
+		IF v_timed_out THEN
+			p_print_msg := 'ERROR:-> TIMED OUT';
+			EXIT Deletes;
+		END IF;
+
+
+
+		-- upon error...
+        EXCEPTION
+
+        	WHEN OTHERS THEN
+
+	        GET STACKED DIAGNOSTICS v_text_var1 = MESSAGE_TEXT,
+	                                v_text_var2 = PG_EXCEPTION_DETAIL,
+	                                v_text_var3 = PG_EXCEPTION_HINT;
+			ROLLBACK;
+	        p_print_msg := 'DELETE FAILED - ERROR:->' || v_text_var1 || '|' || v_text_var2 || '|' || v_text_var3;
+
+	END;
+
+	if exists (
+	   SELECT 1
+	   FROM information_schema.tables
+	   WHERE table_type = 'LOCAL TEMPORARY'
+	   AND table_name = 'deleted_appearance_audit'
+	) then
+		-- clean up the sequence revision numbers
+		DELETE
+		FROM juror_mod.rev_info ri
+		USING deleted_appearance_audit daa
+		WHERE ri.revision_number = daa.revision;
+
+		drop table deleted_appearance_audit;
+	end if;
+
+	if exists (
+		   SELECT 1
+		   FROM information_schema.tables
+		   WHERE table_type = 'LOCAL TEMPORARY'
+		   AND table_name = 'deleted_juror_audit'
+		) then
+			-- clean up the sequence revision numbers
+			DELETE
+			FROM juror_mod.rev_info ri
+			USING deleted_juror_audit dja
+			WHERE ri.revision_number = dja.revision;
+
+			drop table deleted_juror_audit;
+		end if;
+
+	if exists (
+	   SELECT 1
+	   FROM information_schema.tables
+	   WHERE table_type = 'LOCAL TEMPORARY'
+	   AND table_name = 'deleted_financial_audit'
+	) then
+			-- clean up the financial audit details
+			DELETE
+			FROM juror_mod.financial_audit_details_appearances fada
```

Comment:
Have we not already deleted from this table on line 56? should we be deleting from financial_audit_details instead (not the appearances one)?

Reply:
correct! well spotted as I didn't spot that in the original script

Reply:
changed it from financial_audit_details_appearances to financial_audit